demos/toolbar/index.html

```diff
@@ -94,9 +95,9 @@
                 <span class="mdc-toolbar__title">Toolbar</span>
               </section>
               <section class="mdc-toolbar__section mdc-toolbar__section--align-end" role="toolbar">
-                <a href="#" class="material-icons mdc-toolbar__icon" aria-label="Download" alt="Download">file_download</a>
-                <a href="#" class="material-icons mdc-toolbar__icon" aria-label="Print this page" alt="Print this page">print</a>
-                <a href="#" class="material-icons mdc-toolbar__icon" aria-label="Bookmark this page" alt="Bookmark this page">bookmark</a>
+                <a href="#" class="material-icons mdc-toolbar__icon mdc-ripple-surface demo-toolbar-surface" data-mdc-ripple-is-unbounded aria-label="Download" alt="Download">file_download</a>
```

Comment:
Should we call `@include mdc-states(text-primary-on-primary)` from within the CSS for `mdc-toolbar__icon`? And then remove this `mdc-ripple-surface` from the demo?

Reply:
I can probably add ripple styles to `mdc-toolbar__icon` if it's safe to assume all toolbar icons are intended to be interactable (which I think is the case judging by the spec).

If I do that, I should probably also add logic to toolbar's JS to instantiate ripples for its icons.

Reply:
I think we should do that.